charts/kafka-operator/crds/operator-kafka-topic-crd.yaml

```diff
@@ -2,14 +2,12 @@ apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    certmanager.k8s.io/inject-ca-from: {{ .Release.Namespace }}/{{ include "kafka-operator.fullname" . }}-server-cert
```

Comment:
Not sure it's safe to remove this line, i cannot be replaced with a static value.

Reply:
should be fine